docs/testrunner-toolkit.md

```diff
@@ -41,6 +41,56 @@ Furthermore, the framework and browser version support depends on the Sauce Labs
 Each docker image tag is the 'latest' image that supports the specific framework version
 :::
 
+<!--CLOUD FRAMEWORKS GO HERE. This markdown is generated from the test-composer project. Do not edit manually -->
+<!--START_AUTO_GENERATED_TABLE-->
+
+### Supported frameworks in Sauce Cloud
+<Tabs
+  defaultValue="cypress"
+  values={[
+    {label: 'Cypress', value: 'cypress'},
+    {label: 'Playwright', value: 'playwright'},
+    {label: 'TestCafe', value: 'testcafe'},
+    {label: 'Puppeteer', value: 'puppeteer'},
+  ]}>
+  
+<TabItem value="cypress">
+
+|Cypress Version|Supported Platforms|Supported Browsers|
+|-----|-----|-----|
+|5.7.0|Windows 10|Chrome, Firefox, MicrosoftEdge|
+|0.2.3|Windows 10|Chrome, Firefox, MicrosoftEdge|
+
+</TabItem>
+<TabItem value="playwright">
+
+|Playwright Version|Supported Platforms|Supported Browsers|
+|-----|-----|-----|
+|1.7.2|Windows 10|Playwright Chromium, Playwright Firefox, Playwright Webkit|
```

Comment:
Don't think we need to enumerate the browsers here with the `playwright` prefix (despite it being the embedded browsers that come with playwright).


Also, that's the runner version listed there, not the playwright version 🤔